_utils/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
Added a .gitignore here since it seems like `.env` is a build artifact and we don't want it in GitHub. But I can merge this into the parent .gitignore if that seems more kosher.